easybuild/easyconfigs/o/OpenMM-PLUMED/OpenMM-PLUMED-1.0-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'OpenMM-PLUMED'
+version = '1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/openmm/openmm-plumed/'
+description = """This project provides a connection between OpenMM and
+PLUMED. It allows you to bias or analyze an OpenMM simulation based on
+collective variables."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+# https://github.com/openmm/openmm-plumed/archive/refs/tags/v1.0.tar.gz
+github_account = 'openmm'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['97cbffab7a452a3c8f0b00c301c9a796b752ee73e30dc5c1b2435678dd7c73ed']
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('SWIG', '4.0.1'),
+]
+dependencies = [
+    ('Python', '3.8.2'),
+    ('OpenMM', '7.4.2', versionsuffix),
+    ('PLUMED', '2.6.0', versionsuffix),
+]
+
+separate_build_dir = True
+
+configopts = '-DPLUMED_INCLUDE_DIR="$EBROOTPLUMED/include/plumed" '
+configopts += '-DPLUMED_LIBRARY_DIR="$EBROOTPLUMED/lib" '
+configopts += '-DOPENMM_DIR="$EBROOTOPENMM" '
+
+installopts = ' && make -f python/CMakeFiles/PythonInstall.dir/build.make python/PlumedPluginWrapper.cpp'
+installopts += ' && cd python && python setup.py build && python setup.py install --prefix=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ['include/PlumedForce.h', 'include/PlumedKernels.h',
+              'lib/libOpenMMPlumed.%s' % SHLIB_EXT,
+              'lib/python3.8/site-packages/openmmplumed.py'],
```

Comment:
@zarybnicky Please avoid hardcoding the Python version here, by using the `%(pyshortver)s` template:

```python
'lib/python%(pyshortver)s/site-packages/openmmplumed.py'
```